subprojects/inline/inline.gradle

```diff
@@ -0,0 +1,18 @@
+description = "Mockito preconfigured inline mock maker (intermediate and to be superseeded by automatic usage in a future version)"
+
+ext {
+    artifactName = 'mockito-inline'
+    bintrayAutoPublish = true
+    bintrayRepo = 'maven'
+    mavenCentralSync = false
+}
+
+apply from: "$rootDir/gradle/java-library.gradle"
+
+dependencies {
+    compile project.rootProject
```

Comment:
When you run build, can you inspect the pom files generated somewhere under the build dir, and make sure the pom has dependencies? We hit this problem in the past twice so far. I will add validation for this caveat at some point.